### PR TITLE
Automatically exclude incomplete dates ariaAOIassist

### PR DESCRIPTION
**Pull request by rzinke**

### CODE REVIEW

**Comment by dbekaert** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity increasing per file
==============================
- tools/bin/ariaAOIassist.py  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/bekaertdavid/ARIA-tools/pullRequest?prid=5801843&bid=18993505)

**Comment by dbekaert** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity increasing per file
==============================
- tools/bin/ariaAOIassist.py  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/aria-tools/ARIA-tools/pullRequest?prid=5801844&bid=18993506)

File: tools/bin/ariaAOIassist.py [hunk 1]
```diff
@@ -488,6 +503,67 @@ def __polygonFromFrame__(self,frame):
...
+            self.checkContinuity(removeIncompleteDates=True)
+
+        # Check that some valid dates remain
+        errMessage='''
+No valid dates were found to meet all spatial and temporal criteria.
+Suggest to narrow the latitude bounds using the --lat_bounds \'min max\' option, or
+exclude dates using the --exclude_dates option.
+Check the {}_lat_extents.eps file for a visual aid using the --flag_partial_coverage option.
+        '''.format(self.trackCode)
+        assert self.nRemainingDates>0, errMessage
```
Comment (dbekaert) [inline tools/bin/ariaAOIassist.py:524]: Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/gh/aria-tools/ARIA-tools/pullRequest?prid=5801844)

**Comment by dbekaert** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- tools/bin/ariaAOIassist.py  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/aria-tools/ARIA-tools/pullRequest?prid=5801844&bid=18993506)

**Comment by dbekaert** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- tools/bin/ariaAOIassist.py  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/bekaertdavid/ARIA-tools/pullRequest?prid=5801843&bid=18993505)

File: tools/bin/ariaAOIassist.py [hunk 2]
```diff
@@ -32,8 +32,8 @@ def createParser():
             help='Specify directory to deposit all outputs. Default is local directory where script is launched.')
     parser.add_argument('-t', '--tracks', dest='tracks', type=str, default='all',
             help='Include only specified track number in results. Can be multiple, separated by spaces. Default : All')
-    parser.add_argument('-l', '--lat_bounds', dest='latBounds', type=str, default='-75 75',
-            help='Specify a search for only frames that fall within these lat bounds. Default : -75 75')
+    parser.add_argument('-l', '--lat_bounds', dest='latBounds', type=str, default=None,
+            help='Specify a search for only frames that fall within these lat bounds. Default : None')
```
Comment (dbekaert) [inline tools/bin/ariaAOIassist.py:36]: @rzinke can you rebase your PR. Im not sure, but this seems to undo what you had before.

Comment (rzinke) [reply tools/bin/ariaAOIassist.py:36]: Hi David, I meant to remove the default values for lat_bounds. Now there should be no default bounds. 
Could you give this a try with your data to confirm?

**Pull request merged**